src/main/java/code401challenges/linkedlist/LinkedList.java

```diff
@@ -56,4 +51,67 @@ public String toString(){
         }
         return allValues.toString();
     }
+
+
+
+    public void append(int value){
+

```

Comment:
- Check if the list is empty
     - if so, the LL head = new Node
     - if not:
            - iterate through the list and find the last node
            - make the last Node's.next = new Node